cvss2/vector.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cvss2
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	partSeparator   = "/"
+	metricSeparator = ":"
+)
+
+// Vector represents a CVSSv3 vector, holds all metrics inside (base, temporal and environmental)
+type Vector struct {
+	BaseMetrics
+	TemporalMetrics
+	EnvironmentalMetrics
+}
+
+// String returns this vectors representation as a string
+// it shouldn't depend on the order of metrics
+func (v Vector) String() string {
+	var sb strings.Builder
+	fmt.Fprint(&sb, "(")
```

Comment:
So, you define `partSeparator` and `metricSeparator` but not `prefix` and `suffix`? :) Just nitpicking.

Reply:
added {pre,suf}fix